src/go/configgenerator/filtergen/cors.go

```diff
@@ -0,0 +1,35 @@
+package filtergen
```

Comment:
NIT: copyright header? 

Ditto for the remaining files.

Reply:
Done